lib/ovh.es6.js

```diff
@@ -1,6 +1,6 @@
 /**
- * Copyright (c) 2014 OVH SAS
- * Copyright (c) 2012 - 2013 Vincent Giersch
```

Comment:
Yes. Only because this code was initially did on personal time by @gierschv and then take into ovh world. So the first copyright (2012-2013) need to be keep.